### PR TITLE
Render dotnet-new-install for template packages

### PR DESCRIPTION
**Pull request by am11**
With this change, the donet CLI template packages will show the `dotnet-new-install` command.

image

Addresses https://github.com/NuGet/NuGetGallery/issues/5216

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -28,13 +28,27 @@
     {
         packageManagers = new PackageManagerViewModel[]
         {
-            new PackageManagerViewModel(".NET CLI")
+            new PackageManagerViewModel(".NET CLI Tool")
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: /cc @nkolev92

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 2]
```diff
@@ -28,13 +28,27 @@
...
+    {
+        packageManagers = new PackageManagerViewModel[]
+        {
+            new PackageManagerViewModel(".NET CLI Template")
+            {
+                Id = "dotnet-cli",
+                CommandPrefix = "> ",
+                InstallPackageCommand = string.Format("dotnet new --install {0}::{1}", Model.Id, Model.Version),
+                AlertLevel = AlertLevel.Info,
+                AlertMessage = "This package contains a <a href='https://aka.ms/dotnet-new'>.NET Core Template Package</a> you can call from the shell/command line.",
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: I think a better link may be: 
https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates

Thoughts @karann-msft, @anangaur ?

Comment (anangaur) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Seems like the right place but better to get the URL from the PM of the product. @KathleenDollard 😄

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (nkolev92) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: I don't have a preference either way. 

I'd let @anangaur, @karann-msft , @kathleenDollard chime in.

Comment (nkolev92) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: I think leaving it .NET CLI is inline with the rest of the tabs though. 
The tab represents the tool not the action.

Comment (am11) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Fix pushed in next update.

**Pull request merged**